common/common.go

```diff
@@ -46,6 +47,70 @@ func Run() {
 			WithPolling(wait.DefaultInterval).
 			Should(Succeed())
 	})
+
+	It("has all the control-plane nodes running", func() {
+		values := &application.ClusterValues{}
+		err := Framework.MC().GetHelmValues(Cluster.Name, Cluster.Namespace, values)
+		Expect(err).NotTo(HaveOccurred())
+
+		Eventually(wait.Consistent(checkControlPlaneNodesReady(wcClient, values), 12, 5*time.Second)).
+			WithTimeout(wait.DefaultTimeout).
+			WithPolling(wait.DefaultInterval).
+			Should(Succeed())
+	})
+
+	It("has all the worker nodes running", func() {
+		values := &application.ClusterValues{}
+		err := Framework.MC().GetHelmValues(Cluster.Name, Cluster.Namespace, values)
+		Expect(err).NotTo(HaveOccurred())
+
+		Eventually(wait.Consistent(checkWorkerNodesReady(wcClient, values), 12, 5*time.Second)).
+			WithTimeout(wait.DefaultTimeout).
+			WithPolling(wait.DefaultInterval).
+			Should(Succeed())
+	})
+}
+
+func checkControlPlaneNodesReady(wcClient *client.Client, values *application.ClusterValues) func() error {
```

Comment:
Would it make sense to move these functions `checkControlPlaneNodesReady` and `checkWorkerNodesReady` to `clustertest`? I see them as something that the framework provides for the end users, that could be used in different suites.

Reply:
+1 to what Jose suggests

Reply:
The fremework already provides the generic `AreNumNodesReady` and `AreNumNodesReadyWithinRange`, so I wouldn't put these ones there because they're too specific and they're used currently in one place.
I think we can review this again later when we have more tests and we know what we want to expose.